### ISSUE TITLE
how to show always the mobile menu

### ISSUE THREAD

**Issue by nadjak77** [gantry/gantry5#3164]
Hi,
how can I show always the mobile menu?
Empty the "Mobile menue" in the configuration does not work.
Thank you

**Comment by nadjak77** [comment]
Hi, but then only the icon appears. The functionality with the offcanvas does not work. I think I have to change some breakpoints for the script too

**Comment by blizam** [comment]
Not true... at least not true with Nucleus/Helium or any RocketTheme... perhaps you have some caching you need to clear when changing that value

**Comment by nadjak77** [comment]
Ah! I found the fault: I set the mobile menue breakpoint to 80rem. This is to low. I changed it to 120rem and now it is working. Thank you.
